.env.example

```diff
@@ -1,5 +1,6 @@
 PORT=3001
 MONGO_URI=mongodb://localhost:27017/leleka
 JWT_SECRET=superdupersecret
+NODE_ENV="development"
```

Comment:
it's default value, so it's not necessary to add

Reply:
delete